docs/source/events/index.rst

```diff
@@ -1,5 +1,6 @@
-Event logging and Telemetry
-===========================
+patch-1
```

Comment:
Looks like the conflict hasn't quite been fixed as there are remnants of hte branch names.

The heading also needs to keep the `====` underline.